### DISCUSSION TITLE
将linux的zip打包换为appimage打包

### DISCUSSION THREAD

**Discussion by FTS427** [MCSLTeam/MCSL2#119] [功能建议] [answered]
我认为MCSL2可以尝试使用appimage打包替换掉现在的zip打包，这将会使现在将zip解压后在一堆文件里找主程序的这种麻烦的情况消失
当然，我也认为这对在linux上移动和更新MCSL2上会有帮助，移动单个appimage包要比移动一个文件夹更轻便得多

**Comment by lxhtt** [comment, marked as answer]
与插件系统冲突，无法解决。
